about.html

```diff
@@ -0,0 +1,139 @@
+                    Naturally, the people took advantage and the simple yet quintessential combination of roasted
+                    plantain and perfectly seasoned mackerel enveloped in a mouthwatering peppery palm oil sauce, also
+                    known as bole and fish, was birthed. Bole refers to the roasted plantains and the fish which is
+                    usually mackerel is always inexpensive. <br>Today, every street corner from Abuloma to Elekahia
+                    boasts colourful umbrellas shielding ankara-clad women engulfed in the pungent,scent of smoked fish
+                    and sweet palm oil. The beautiful contrast of white flesh against the blackened mackerel skin, the
+                    vibrant red pepper sauce perfectly binds together with just enough heat and spice to make a grown
+                    man cry.
+                    Within only a few minutes of waiting patiently for ones expectations to be surpassed, the richness
+                    and fullness of this very popular road-side meal can be enjoyed, complete with newspaper wrapping
+                    and black nylon bag for added safe keeping.
+
+                    Port Harcourt is THE capital of bole and fish. History, culture and numerous satisfied stomachs can
+                    testify to this. However, should you still need proof, next time you are in the city, be sure to
+                    include a quick stop at D-Line and experience first-handedly, the authentic and flavoursome dish
+                    served from the Headquarters of Bole.
+                </p>
+            </div>
+        </div>
+        <section class="history">
+            <div class="main-head">
+                <h2>Highlights of Past Bole Festival</h2>
+                <hr>
+                <p> Past Bole festivals both in lagos and in Abuja</p>
+            </div>
+            <div class="past-img-desk">
+                <div class="past-imgs">
+                    <h2>Lagos Bole Festival 2003 </h2>
+                </div>
+                <div class="past-imgs2">
+                    <h2>Abuja Bole Festival 1990 </h2>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <section class="partners" id="sponsors">
+            <h4>
+                Sponsored By
+            </h4>
+            <hr>
+            <div class="partners-logo">
+                <img src="./img/GTBank_logo.png" alt="gtbank">
+                <img src="./img/coker.png" alt="cocacola">
+                <img src="./img/Government_of_Rivers_State_logo.png" alt="Rs-govt">
+                <img src="./img/Rhythm_937_FM.png" alt="93.7fm">
+                <img src="./img/McDonald's_logo.png" alt="McDonald">
+            </div>
+        </section>
+        <section class="company">
+            <img src="./img/bolefest.png" alt="bole festival">
+            <p>2023 Bole Festival Port Harcourt <br>some Rights Reserved.</p>
+        </section>
```

Comment:
- [ ]  Your about page footer should be similar to the home footer,  please make sure that both page have same structured footer part. please don't forget that its footer is supposed to be dark, while on home page its white. 
![image](https://user-images.githubusercontent.com/43877689/227659197-254cb4dc-e2ea-4cbe-a2bb-56c874a8a5b3.png)